### DISCUSSION TITLE
VIP door entry systems

### DISCUSSION THREAD

**Discussion by dotgrid** [gicamm/homeassistant-comelit#46] [General]
Thanks for releasing this!

I'm looking to connect my flat's VIP door entry system (with two-way secure video) to Home Assistant, and then to HomeKit. 

Could you point me in the first direction of how to start this project? What Comelit hardware I'd need?

**Comment by gicamm** [comment]
Hi,
The integration works with Comelit Hub 20003150.
You need to understand if your system is compatible.
